protocol-marketplace/caffe-mnist.yaml

```diff
@@ -0,0 +1,58 @@
+protocolVersion: 2
```

Comment:
Isn't "marketplace/v2" a more aligned folder path to echo with "*/v2" in PR #2260?

Reply:
Because the original marketplace serve the whole dir, so I added a new separate directory.

Reply:
renamed to `marketplace-v2`